index.html

```diff
@@ -69,12 +72,235 @@ <h1 class="col col-11 col-lg-5 aligne-self-center"> glade to see you</h1>
+                        <span class="att2">2015</span>
+                    </div>
+                    <span class="card-text">A daily selection of privately 
+                        personalized reads; no accounts or sign-ups required.</span>
     
+                    <div class="tag-section">
+                        <div class="tag-div">
+                            <span class="tag">html</span>
+                        </div>
+                        <div class="tag-div">
+                            <span class="tag">css</span>
+                        </div>
+                        <div class="tag-div">
+                            <span class="tag">javaScript</span>
+                        </div>
+                    </div>
+                    <a href="#" class="button">See Project</a>
+            
+            </div>
+            
+            
+        </div>
+        <div class="grid-card">
+            
+            <img class="card-img" src="./assets/Snapshoot4.png" alt="Card image">
+            <div class="card-body">
+                <h1 class="card-title">Multi-Post Sorties</h1>
+                    <div class="card-attribut">
+                        <span class="att1">CANOPY</span>
+                        <img src="./assets/Counter.png" class="dot" alt="dot">
+                        <span class="att2">Back End Dev</span>
+                        <img src="./assets/Counter.png" class="dot" alt="dot">
+                        <span class="att2">2015</span>
+                    </div>
+                    <span class="card-text">A daily selection of privately personalized 
+                        reads; no accounts or sign-ups required.</span>
     
-    
-   
+                    <ul class="tag-section">
+                        <li class="tag-div">
+                            <span class="tag">html</span>
+                        </li>
+                        <li class="tag-div">
+                            <span class="tag">css</span>
+                        </li>
+                        <li class="tag-div">
+                            <span class="tag">javaScript</span>
+                        </li>
+                    </ul>
+                    <a href="#" class="button">See Project</a>
+            </div>
+            
+            
+        </div>
+    </section>
+
+    <section class="about-me">
+        <h1 class="card-title">About</h1>
+        <h1 class="card-title">Myself</h1>
+
+        <p class="card-text">
+            Hello I’m a software developer! 
+            I can help you build a product , 
+            feature or website Look through 
+            some of my work and experience! 
+            If you like what you see and have a 
+            project you need coded, don’t hestiate to contact me.
+        </p>
+        <h3 class="let">LET'S CONNECT</h3> <br>
+                <div class="link-list">
+                    <li class="link">
```

Comment:
- [ ] You still have linter errors caused by this line. 

`<li>` tags should have `<ul>` parents. You can easily fix this error with nesting your `<li>` element with `<ul>`